how-we-work/mozmeao.md

```diff
@@ -0,0 +1,142 @@
+
+### In Progress
+
+Tasks that are actively being worked on. This can include (but is not limited to) analysis, development and/or troubleshooting. We should not have more than our WIP number of cards in this column. See the WIP section [TODO: link] for more info.
+
+### Review
+
+Tasks that are waiting for QA, PR review, discussion, and/or merging. Once a card is in this column, you can pull a new card off the Queued column.
+
+### Complete
+
+The task is complete, it is _live_ and no additional work is required. Cards left in this column can be added to the biweekly blog post [TODO]. Once the blog post is written, move cards to `dev/null`.
+
+### dev/null
+
+This is where cards go to collect dust, as their doesn't seem to be an easy way to hide completed cards.
+
+## Work-in-progress limits
+
+Each engineer should have between 1-3 issues that they are working on, which we'll call the `WIP limit`. For a team of 4 backend engineers/SRE's, let's start with 4 * 3. While this limit is not strictly enforced, exceeding it will cause more context switching and possibly slow down other work.
+
+The title of for the `In progress` column should include `(limit 12)`, or whatever limit we decide is appropriate.
+
+
+## Meetings
+
+### Weekly board review meeting
+
+We have a weekly project review meeting on Wednesdays at [TODO]. 
+The goal of this meeting is to review our project board, replenish the `Queued` column of the board with new work, and to identify blockers or at-risk tasks.
+
+#### Process:
+
+- Start with the top of the `Review` column, ensure that each issue is receiving appropriate review. If the issue has already been resolved/merged/closed, move to the **top** of the `Complete` column.
+- Next, review each card in the `In Progress` column to ensure the issue isn't blocked. Blocked issues should be updated with an appropriate note. When an engineer has submitted a PR, the card should be moved to the **top** of the `Review` column. Pull requests are _usually_ reviewed within 1 business day, and don't need to wait until the weekly triage meeting to be moved to the `Review` column.
+    - Let's make sure the `In Progress` column is obeying the WIP limit.
+- Next, review the `Queued` column starting from the top. The top represents the highest priority queued tasks that engineers should work on next.
+    - are there any cards in this column that have been deferred? If so, do they still belong on this project board?
+    - move cards from `Queued` to `In Progress`, and assign one or more engineers to work on the issue.
+- Project owners will add cards to this column for their respective projects. We should strive to keep ~2 weeks of work in the `Queued` column.
+- Once the board has been reviewed, decide who will write the bi-weekly blog post and assign a card in the `Queued` column.
+
+
+### Standups
+
+***DRAFT***
+
+We'll run a standups on Monday, Tuesday, Thursday and Friday for ***10 minutes***. Time TBD.
```

Comment:
I think we should consider nixing the friday in-person standup. Two reasons for this:

1. No meeting friday.
2. Generally this would be late friday afternoon for Giorgos, we should avoid that.